clients/lsp-terraform.el

```diff
@@ -128,6 +146,11 @@ Defaults to side following treemacs default."
   (when lsp-terraform-ls-enable-show-reference
     '((experimental . ((showReferencesCommandId . "client.showReferences"))))))
 
+(defun lsp-terraform-ls--init-options ()
+  "Construct initialization options for the lanague server."
```

Comment:
typo "lanague"

Reply:
Ouch, missed it. Will open a follow up PR, thanks!